docs/admin.rst

```diff
@@ -294,6 +294,47 @@ web server to apply the changes:
 
   sudo service apache2 restart
 
+.. _submission-cleanup:
+
+Cleaning up deleted submissions
+&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&&
+
+When submissions are deleted through the web interface, their database
+records are deleted and their encrypted files are securely wiped. For
+large files, secure removal can take some time, and it's possible,
+though unlikely, that it can be interrupted, for example by a server
+reboot. In older versions of SecureDrop this could leave a submission
+file present without a database record.
+
+As of SecureDrop 1.0.0, automated checks send OSSEC alerts when this
+situation is detected, recommending you run ``./manage.py
+list-disconnected-fs-submissions`` to see the files affected. (As with
+any ``manage.py`` usage, you would first SSH to the application
```

Comment:
This should be a series of steps, ie:
- log into ye olde Workstationne
- ssheth to yon app
- begin the masquerade! by putting on the guise of the lord of www-data
- maketh a holy pilgrimage to the heart of the world at /var/www/securedrop
- etc.

Having all the steps in a paragraph is a bit harder to read.

Reply:
:smile: Done.